src/bakery/Client.cpp

```diff
@@ -4,10 +4,10 @@
 
 #include "Client.h"
 
-Client::Client(int id, const string &name, const Vertex &address, const Time &scheduledTime, int breadQuantity) {
+Client::Client(int id, const string &name, Vertex* vertex, const Time &scheduledTime, int breadQuantity) {
     this->id = id;
     this->name = name;
-    this->address = address;
+    this->vertex = vertex;
     this->scheduledTime = scheduledTime;
     this->breadQuantity = breadQuantity;
     this->realTime = Time(0);
```

Comment:
Isn't this going to be changed to a position? (instead of a vertex)

Reply:
I changed it to a vertex, just because we were doing the verification on the input file about being a valid Vertex, so we had to search through the vertexSet anyway and since the Client stored both the Position and the vertexID, I thought we could just store the pointer to the Vertex (that stores both the Position and the vertexID), since its 1 variable instead of 2 and might simplify it. Is there a reason why we shouldn't?